.gitignore

```diff
@@ -104,3 +104,7 @@ typespec-tests/package-lock.json
 
 # ApiView properties files
 /typespec-tests/src/main/resources/META-INF/*apiview_properties.json
+
+# typespec-extension
+/typespec-extension/src
+/typespec-extension/generator
```

Comment:
what is `/typespec-extension/generator` for?

Reply:
These are the places we copy JAR and TypeScript code into typespec-extension (from core) -- so that typespec-extension can still pack to typespec-java

Since they are copied, we don't want to commit them.